ducktape/cluster/vagrant.py

```diff
@@ -46,6 +47,7 @@ def __init__(self, *args, **kwargs):
                 # It is OK if file is not found. Call vagrant ssh-info to read the cluster info.
                 pass
 
+        cluster_json: Optional[dict] = None
```

Comment:
this looks like a bug. cluster_json is set a few lines above and overwritten here?

the type annotation not he parent constructor is all that should be needed. is a static checker complaining?

Reply:
lol, yep.  Warning (something like `cluster_json` may be unbound) went away--but I think this (or an empty dict initialization) needs to move up in this function.

Reply:
"fixed a warning, added a bug" :laughing: